source/reference_guide/concepts_info/concepts/attack_choice_system.rst

```diff
@@ -0,0 +1,21 @@
+.. _concept_attack_choice_system:
+
+Attack choice system
+============================================
+
+.. card::
+   :class-card: sd-mt-3
+   :class-header: header-2-light
+
+   Description
+   ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+   Once a faction has positive attack resources, it will call chooseAttack to launch an attack. Target selection is handled with findAttackTargets, which weights potential attacks depending on target value and threat. Main differences:
+
+   - Lower-rated attacks can be chosen, using the weight as a probability.
+   - Attacks against rebel targets outside mission distance are now permitted, although chance is reduced.
+   - Danger of flying over other enemy locations is taken into account.
+   - Threat of other nearby locations is taken into account, not just the target.
+   - Distance has no effect (other than the hard air distance cap), but nearby land bases are beneficial.
```

Comment:
Changed, air distance now has some effect, although less than previously.

Reply:
can you tell me which impact or rephrase that more friendly for people? i can't follow what you wanna tell me with that

Reply:
Distance from air bases has less effect, with nearby land bases being more important.

Reply:
replaced sentence